dialogue-apache-hc5-client/src/test/java/com/palantir/dialogue/hc5/ApacheHttpClientChannelsTest.java

```diff
@@ -124,6 +125,38 @@ public void metrics() throws Exception {
         }
     }
 
+    @Test
+    public void countsUnknownHostExceptions() throws Exception {
+        ClientConfiguration conf = TestConfigurations.create("http://unused");
+
+        try (ApacheHttpClientChannels.CloseableClient client =
+                ApacheHttpClientChannels.createCloseableHttpClient(conf, "testClient")) {
+
+            TaggedMetricRegistry metrics = conf.taggedMetricRegistry();
+
+            assertThat(DialogueClientMetrics.of(metrics)
+                            .connectionResolutionError("testClient")
+                            .getCount())
+                    .isZero();
```

Comment:
pedantic nit: We can hold the `Meter` instance and check it again on line 153 rather than looking it up a second time.